wiki/Beatmap_Editor/Timing/id.md

```diff
@@ -1,19 +1,25 @@
----
-outdated: true
----
+# Tab timing
 
-# Timing
+*Kunjungi juga: [Bagaimana cara mengatur waktu lagu](/wiki/Guides/How_to_Time_Songs)*
 
-Timing untuk sebuah beatmap terdiri dari offset lagu dan BPM(Beats Per Minute, atau ketukan per menit). BPM lagu menunjukkan seberapa cepat sebuah lagu. Offset adalah jumlah waktu di antara awal dari file mp3 dan pembukaan lagu dalam sebuah lagu.
+![Cuplikan tab timing di dalam editor](img/Timing_base.jpg)
 
-Fungsi dari timing yang benar adalah agar linimasa di dalam editor sejajar dengan musiknya. Jika timingnya salah, maka linimasa tidak akan sesuai dengan lagu. Oleh karena itu, kamu tidak dapat menempatkan objek yang diklik dengan akurat, dan akan menjadi sangat sulit untuk me-map sejajar dengan musiknya.
+**Tab timing** di dalam [beatmap editor](/wiki/Beatmap_Editor) memungkinkan para mapper mengubah pengaturan timing dari sebuah [beatmap](/wiki/Beatmap). Timing biasanya merupakan sebuah langkah pertama dalam membuat sebuah beatmap.
 
-## Pembagian Timing
+Timing terdiri dari sebuah [offset](/wiki/Beatmapping/Offset) lagu, [beats per minute (BPM)](/wiki/Beatmapping/Beats_per_minute), dan [meter](https://en.wikipedia.org/wiki/Metre_(music)). Beatmap yang mempunyai timing akurat diperlukan bagi pemain untuk menjaga akurasi saat bermain. Beatmap dengan timing yang tidak tepat, tidak dapat melewati [prosedur beatmap ranking](/wiki/Beatmap_ranking_procedure).
```

Comment:
```suggestion
Timing terdiri dari sebuah [offset](/wiki/Beatmapping/Offset) lagu, [beats per minute (BPM)](/wiki/Beatmapping/Beats_per_minute), dan [meter](https://en.wikipedia.org/wiki/Metre_(music)). Beatmap yang mempunyai timing akurat diperlukan agar pemain dapat menjaga akurasi pada saat bermain. Beatmap dengan timing yang tidak tepat tidak dapat melewati [prosedur beatmap ranking](/wiki/Beatmap_ranking_procedure).
```

Reply:
it'd be better if we could rework the sentence to some extent, another thing that i had on my mind is to add something to the ```tidak dapat melewati``` part. for example, ```tidak akan bisa melewati``` / ```tidak akan dapat melewati```.